controller/query.go

```diff
@@ -0,0 +1,210 @@
+package controller
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/fabric8-services/fabric8-wit/app"
+	"github.com/fabric8-services/fabric8-wit/application"
+	"github.com/fabric8-services/fabric8-wit/errors"
+	"github.com/fabric8-services/fabric8-wit/jsonapi"
+	"github.com/fabric8-services/fabric8-wit/login"
+	"github.com/fabric8-services/fabric8-wit/query"
+	"github.com/fabric8-services/fabric8-wit/rest"
+	"github.com/fabric8-services/fabric8-wit/space"
+	"github.com/goadesign/goa"
+	"github.com/prometheus/common/log"
```

Comment:
Wrong import path.

Reply:
thanks @baijum   fixed in https://github.com/fabric8-services/fabric8-wit/pull/1824/commits/50108ded08fe6e3b3a39474e319cf43c6c30f98b